flink-clients/src/main/java/org/apache/flink/client/program/PackagedProgram.java

```diff
@@ -90,6 +91,9 @@
 
 	private SavepointRestoreSettings savepointSettings = SavepointRestoreSettings.none();
 
+	// Flag indicating whether the job is a Python job.
```

Comment:
/**
	 * Flag indicating whether the job is a Python job.
	 */
to keep consist of other comments.